packages/relay/src/RelayServer.ts

```diff
@@ -428,70 +435,19 @@ returnValue        | ${viewRelayCallRet.returnValue}
     return signedTx
   }
 
-  async intervalHandler (): Promise<void> {
-    const now = Date.now()
-    let workerTimeout: Timeout
-    if (!this.config.devMode) {
-      workerTimeout = setTimeout(() => {
-        const timedOut = Date.now() - now
-        this.logger.warn(chalk.bgRedBright(`Relay state: Timed-out after ${timedOut}`))
-      }, this.config.readyTimeout)
-    }
-    let gotBlock = false
-    // eslint-disable-next-line @typescript-eslint/return-await
-    return this.contractInteractor.getBlockNumber()
-      .then(
-        blockNumber => {
-          gotBlock = true
-          if (blockNumber < this.config.coldRestartLogsFromBlock) {
-            throw new Error(
-              `Cannot start relay worker with coldRestartLogsFromBlock=${this.config.coldRestartLogsFromBlock} when "latest" block returned is ${blockNumber}`)
-          }
-          if (blockNumber > this.lastScannedBlock) {
-            return this._workerSemaphore.bind(this)(blockNumber)
-          }
-        })
-      .catch((e) => {
-        this.emit('error', e)
-        const error = e as Error
-        this.logger.error(`error in worker: ${error.message} gotBlockNumber: ${gotBlock} ${error.stack}`)
-        this.setReadyState(false)
-      })
-      .finally(() => {
-        clearTimeout(workerTimeout)
-      })
-  }
-
   start (): void {
     this.logger.debug(`Started polling for new blocks every ${this.config.checkInterval}ms`)
```

Comment:
new logging are better (info for one-time items, debug for repeated, non-critical).
this `debug` should match the `stop` conditions, and be `info`

Reply:
Ok (though I didn't touch that line here...)